schema/pharmgkbScriptSchema.mcf

```diff
@@ -0,0 +1,203 @@
+Node: dcid:alfredID
```

Comment:
I might rename the files as:

  PharmGKB.mcf
  PharmGKBIDs.mcf

(Either Init-cap camel case or snake-case)